frontend/src/constants.scss

```
@@ -0,0 +1,15 @@
+$prussian-blue: #012e40ff;
+$blue-sapphire: #025e73ff;
+$blue-green: #049dbfff;
+$dark-turquoise: #3bc9d9ff;
+$blizzard-blue: #bbedf2ff;
+$white: white;
+$black: black;
+
+$primary: $blue-green; // Used for main elements that needs to be highlighted and buttons
+$primary-light: $blizzard-blue; // Used for background to text boxes that pops up or need to stand out from white background
+$primary-dark: $blue-sapphire; // Used for sub-headings as a background colour
+$secondary: $prussian-blue; // Used for headers with text color as white
+
+$text-color: $black; // If on dark background, change to white
```

Rather than add a comment, add another variable for this case:
```suggestion
$text-color: $black;
$text-colour-light: $white;  // For use on dark backgrounds
```